Extension/.eslintrc.js

```diff
@@ -57,6 +58,10 @@ module.exports = {
         "@typescript-eslint/triple-slash-reference": "error",
         "@typescript-eslint/type-annotation-spacing": "error",
         "@typescript-eslint/unified-signatures": "error",
+        "@typescript-eslint/no-floating-promises": "error",
+      "no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" } ],
```

Comment:
Can you run the formatter? I'm seeing tab/space differences.

Reply:
👍 

Yeah, I'm seeing them too, my default settings are slightly different, and the `.vscode/settings.json`  in the repo doesn't have  indent sizes specified, so I was going to bring that up in the meeting, and then set it accordingly, and then make sure everything is formatted correctly before coming out of draft mode. 

I'm worried about it completely freaking out and making it look like massive changes due to indenting.

Reply:
It looks like we use 4 spaces for indentation, but I don't know if/how that is configured. Maybe you can set the editor.indentSize to for in .vscode/settings.json?

Reply:
That's what I'd like to do - there are some files that are using tabs and some using a different indent, so before I was going to force it all one way, I wanted to talk about it in the meeting. 

Ideally, it'd be perfectly consistent, and this issue doesn't ever come back.